glib-macros/src/downgrade_enum.rs

```diff
@@ -0,0 +1,74 @@
+use crate::downgrade_fields::{derive_downgrade_fields, DowngradeStructParts};
```

Comment:
Instead of prefixing the filenames, maybe move all this into a downgrade module. E.g. `downgrade::enum`

Reply:
Done